dgraph/cmd/bulk/run.go

```diff
@@ -118,7 +118,9 @@ func init() {
 	flag.Bool("new_uids", false,
 		"Ignore UIDs in load files and assign new ones.")
 	flag.Uint64("force-namespace", math.MaxUint64,
-		"Namespace onto which to load the data. If not set, will preserve the namespace.")
+		"Namespace onto which to load the data. If not set, will preserve the namespace."+
+			" When using this flag to load data into specific namespace, make sure that the "+
```

Comment:
What's the behaviour in case it does have the ACL data?

Reply:
I think it will just sail through into the p directory though it is probably not recommended. The user should rather add rules using dgraph APIs. Maybe, we could check for this during bulk loader.